modules/cms/twig/SnowboardNode.php

```diff
@@ -1,11 +1,13 @@
-<?php namespace Cms\Twig;
+<?php
 
-use Config;
+namespace Cms\Twig;
+
+use Illuminate\Support\Facades\URL;
```

Comment:
Any issues with the "URL" vs "Url"?

Reply:
Not that I have seen - I don't think PHP is case-sensitive in that regard.